extension/src/commands/run.ts

```diff
@@ -1,13 +1,14 @@
-import { getAppHostProject } from '../utils/projects';
+import * as vscode from 'vscode';
 import { getAspireTerminal } from '../utils/terminal';
+import { isWorkspaceOpen } from '../utils/vsc';
 
 export async function runCommand() {
+    if (!isWorkspaceOpen()) {
+        return;
+    }
+
     const terminal = getAspireTerminal();
-    // Show a quick pick to the user to select a project
-    const selectedProject = await getAppHostProject();
 
-    if (selectedProject) {
-        terminal.sendText(`aspire run --project "${selectedProject}"`);
-        terminal.show();
-    }
+    terminal.sendText(`aspire --language ${vscode.env.language} run`);
```

Comment:
Not sure we'll need `--language`? But maybe I haven't thought about it enough.

Reply:
Let’s just make it work without language for now @adamint